modules/nextMillenniumBidAdapter.js

```diff
@@ -24,12 +24,18 @@ export const spec = {
 
     _each(validBidRequests, function(bid) {
       window.nmmRefreshCounts[bid.adUnitCode] = window.nmmRefreshCounts[bid.adUnitCode] || 0;
+      const id = getPlacementId(bid)
+
+      if (bid.sizes && !Array.isArray(bid.sizes[0])) bid.sizes = [bid.sizes]
+      if (!bid.ortb2) bid.ortb2 = {}
+      if (!bid.ortb2.device) bid.ortb2.device = {}
+      bid.ortb2.device.referrer = getWindowTop().document && getWindowTop().document.referrer
```

Comment:
why not use getRefererInfo().page ?

Reply:
I updated it, 
I just didn't know about this...

Thanks!